lib/model/image_sscd.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict
```

Comment:
Is there a compelling reason to persist a separate SSCD file @computermacgyver or @ahmednasserswe ?

Reply:
No I think this file can and should be removed